hybridauth/Hybrid/Auth.php

```diff
@@ -352,6 +352,9 @@ public static function logoutAllProviders() {
 	 * @param string $mode PHP|JS
 	 */
 	public static function redirect($url, $mode = "PHP") {
+		if(!$mode){
+			$mode = 'PHP';	

```

Comment:
I am not sure if this is needed.   Can you explain why it is?

```
<?php 

function redirect($url, $mode = "PHP") {
  echo $mode;
}

redirect('https://test.com/');
?>
```

will output PHP because the second parameter is already set 

$mode = "PHP"

so I do not understand why check it again.